components/shared/publizonPlayer/helper.ts

```diff
@@ -0,0 +1,29 @@
+type TAssetType = {
+  src: string
+  type: "script" | "link"
+}
+
+export const assets: TAssetType[] = [
+  {
+    src: "https://play.pubhub.dk/1.3.0/js/player-kernel.min.js",
+    type: "script",
+  },
+]
+
+export const appendAsset = ({ src, type }: TAssetType) => {
+  if (type === "script") {
+    const scriptElement = document.createElement("script")
+    scriptElement.src = src
+    scriptElement.defer = true
+    scriptElement.async = false
+    scriptElement.type = "module"
+    document.head.appendChild(scriptElement)
+  }
+}
+
+export const removeAsset = ({ src, type }: TAssetType) => {
+  if (type === "script") {
+    const scriptElement = document.querySelector(`script[src="${src}"]`)
+    scriptElement?.remove()
+  }
+}
```

Comment:
Why having two sets of `appendAsset` `removeAsset` functions?
As far as I see we can a common way of doing this in `/lib/helpers/reader-player/...` and share it among the components.
You could consider making your custom hook adding the assets receiving an array of assets - which you could use in both the reader and player?

Reply:
Great idea. I will do that and merge the branch into main afterwards.